src/dotnet-new3/dotnet-new3.csproj

```diff
@@ -1,30 +1,30 @@
 <Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <Description>Template Instantiation Commands for .NET Core CLI.</Description>
-        <TargetFramework>$(NETCoreTargetFramework)</TargetFramework>
-        <OutputType>Exe</OutputType>
-    </PropertyGroup>
+  <PropertyGroup>
+    <Description>Template Instantiation Commands for .NET Core CLI.</Description>
+    <TargetFramework>$(NETCoreTargetFramework)</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
```

Comment:
nit: Intentional indentation changes?

Reply:
no, these changes are intentional - csproj files are not respecting changes at the moment, and we are fixing it when touching the files.